contracts/upgradeable_contracts/erc20_to_erc20/FeeManagerErcToErcPOSDAO.sol

```diff
@@ -13,7 +13,14 @@ contract FeeManagerErcToErcPOSDAO is BlockRewardFeeManager {
     }
 
     function setBlockRewardContract(address _blockReward) external {
-        require(_blockReward != address(0) && isContract(_blockReward) && (IBlockReward(_blockReward).bridgesAllowedLength() != 0));
+        require(_blockReward != address(0) && isContract(_blockReward));
+        bool isBlockRewardContract = false;
```

Comment:
Since it could be a questionable place in the future let's add a commit like "Before store the contract we need to make sure that it is the block reward contract in actual fact, call a specific method from the contract that should return a specific value"

Reply:
Added comments https://github.com/poanetwork/poa-bridge-contracts/pull/197/commits/f0382fb915c5122b9dd15962cf8364d6789fda8f

Reply:
thanks